docs/advanced.md

```diff
@@ -0,0 +1,62 @@
+# Advanced Options
+
+{% hint style="warning" %}
+This is a section detailing the more advanced parts of using Northstar, with some of them having a chance to potentially mess with your game. Setting your level too high, for example, will lock you out of the game until you reset your entire player stats, progress, and loadouts.
+{% endhint %}
+
+## Mod profiles <a href="#profiles" id="profiles"></a>
+
+Profiles are a way to launch a version of Northstar with specific mods enabled, differing per profile. These profiles are seperated by folders that you create yourself, and add your own mods to. This is especially useful if you want to [play vanilla using Northstar](advanced.md#vanilla-on-northstar). When launching using profiles, you should make sure EA/Origin are open before launching as to make sure you encounter less issues.
```

Comment:
Typo, singular form cause one only uses either Origin or EA-App.

```suggestion
Profiles are a way to launch a version of Northstar with specific mods enabled, differing per profile. These profiles are separated by folders that you create yourself, and add your own mods to. This is especially useful if you want to [play vanilla using Northstar](advanced.md#vanilla-on-northstar). When launching using profiles, you should make sure EA/Origin is open before launching as to make sure you encounter less issues.
```

Reply:
Actually, we might even wanna drop the last line as EA/Origin also needs to be open for Northstar to launch properly (unless launching via Steam)